src/gnmi_server/client_subscribe.go

```diff
@@ -117,28 +118,36 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 		return grpc.Errorf(codes.NotFound, "Invalid subscription path: %v %q", err, query)
 	}
 	var dc sdc.Client
-	if target == "OTHERS" {
-		dc, err = sdc.NewNonDbClient(paths, prefix)
-	} else {
-		dc, err = sdc.NewDbClient(paths, prefix)
-	}
 
-	if err != nil {
-		return grpc.Errorf(codes.NotFound, "%v", err)
-	}
+    if target == "OTHERS" {
+            dc, err = sdc.NewNonDbClient(paths, prefix)
+    } else if target == "APP_DB" || target == "CONFIG_DB" || target == "COUNTERS_DB" {
```

Comment:
Can you define this list else where, as "legacy_mode_client" or something like that ?